src/EmotiBitVariants.h

```diff
@@ -0,0 +1,16 @@
+#pragma once
+
```

Comment:
I moved it to a new file, because i think we will be using this information in the SW as well. something like shown below.
![image](https://user-images.githubusercontent.com/31810812/134423608-7d27531a-1ac4-48da-a8e1-1dfdccd3d7e1.png)
I believe, the packet header(message types), will live in emotibit packet.h, but i don't think that SKU types should live there as well. 
So, i moved the SKU stuff in this file, so that it can be used by the firmware ad the SW(in the future) as well